firefox-ios/Client/Frontend/NativeErrorPage/NativeErrorPageViewController.swift

```diff
@@ -89,37 +89,67 @@ final class NativeErrorPageViewController: UIViewController,
 
     private var contraintsList = [NSLayoutConstraint]()
 
-    required init?(
```

Comment:
How come we're removing this?

Reply:
The "required init" function is present in the code. It was shifted to just below the "unsubscribeFromRedux" function.